docs/networks/testnet.md

```diff
@@ -0,0 +1,111 @@
+# Testnet Configuration
+
+The Graph Network's testnet is on Goerli, and network information can be found at https://testnet.thegraph.com/explorer/network.
```

Comment:
Suggest minor update: "The Graph Network's testnet is on Goerli. Goerli network information can be found at https://testnet.thegraph.com/explorer/network."